Tribler/Core/Modules/process_checker.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from Tribler.Core.SessionConfig import SessionStartupConfig
+
+
+LOCK_FILE_NAME = 'tribler.lock'
```

Comment:
Now that I think of it, QApp has some facility to check for it. So it may not be needed after all.